### PR TITLE
Client mitigate log files

### PR DESCRIPTION
**Pull request by CalvinKrist**

### CODE REVIEW

**Comment by CalvinKrist** [comment]
Can partially test using [these commands](https://social.technet.microsoft.com/Forums/windows/en-US/4f56f320-266a-4075-94e7-3664bcb4bd3e/task-history-is-disabled?forum=w7itprogeneral)

File: BLUESPAWN-client/src/util/eventlogs/EventLogs.cpp [hunk 1]
```diff
@@ -306,4 +306,85 @@ namespace EventLogs {
...
+
+		return pProperty->BooleanVal;
+	}
+
+	bool OpenChannel(const std::wstring& channel) {
+		EVT_HANDLE hChannel = NULL;
+		DWORD status = ERROR_SUCCESS;
+		EVT_VARIANT ChannelProperty;
+		DWORD dwBufferSize = sizeof(EVT_VARIANT);
+		DWORD dwBufferUsed = 0;
```
Comment (ION28) [inline BLUESPAWN-client/src/util/eventlogs/EventLogs.cpp]: Codacy Issue found: [Variable 'dwBufferUsed' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5037686)

File: BLUESPAWN-client/src/util/eventlogs/EventLogs.cpp [hunk 2]
```diff
@@ -306,4 +306,85 @@ namespace EventLogs {
...
+				else {
+					if (pProperty)
+						free(pProperty);
+
+					LOG_ERROR(L"EventLogs::IsChannelOpen: realloc failed for channel " + channel);
+					return false;
+				}
+			}
+
+			if (ERROR_SUCCESS != (status = GetLastError())) {
```
Comment (ION28) [inline BLUESPAWN-client/src/util/eventlogs/EventLogs.cpp]: Codacy Issue found: [Variable 'status' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5037686)

File: BLUESPAWN-client/src/util/eventlogs/EventLogs.cpp [hunk 3]
```diff
@@ -217,4 +217,85 @@ namespace EventLogs {
...
```
Comment (ION28) [inline BLUESPAWN-client/src/util/eventlogs/EventLogs.cpp]: Codacy Issue found: [Variable 'status' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5037686)

File: BLUESPAWN-client/src/util/eventlogs/EventLogs.cpp [hunk 4]
```diff
@@ -217,4 +217,85 @@ namespace EventLogs {
...
+
+		return pProperty->BooleanVal;
+	}
+
+	bool OpenChannel(const std::wstring& channel) {
+		EVT_HANDLE hChannel = NULL;
+		DWORD status = ERROR_SUCCESS;
+		EVT_VARIANT ChannelProperty;
+		DWORD dwBufferSize = sizeof(EVT_VARIANT);
+		DWORD dwBufferUsed = 0;
```
Comment (ION28) [inline BLUESPAWN-client/src/util/eventlogs/EventLogs.cpp]: Codacy Issue found: [Variable 'dwBufferUsed' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5037686)

**Pull request merged**